cfme/tests/configure/test_access_control.py

```diff
@@ -1104,7 +1115,7 @@ def nav(appliance):
         [  # Param Set 1
             [  # product_features
                 [['Everything'], False],  # minimal permission
-                [['Everything', 'Settings', 'Tasks'], True]
+                [['Everything', SETTINGS, 'Tasks'], True]
```

Comment:
@digitronik how are the PRT failures not related? I see a failure like 
```
b"message: Could not find the item Everything/Settings in Boostrap tree features_treebox, path: ('Settings', 'Configuration'), cause: Was not found in Everything"
```
which looks like it could point to a fix here?

Reply:
@john-dupuy I added changes in `test_permission_edit` and `test_permissions`. There are not related with this changes. 

but let me check if i can include fix in same PR.

Reply:
as i see `role_crud` fixed by the pr https://github.com/ManageIQ/integration_tests/pull/9741/files